pkg/cloud/vsphere/services/kubeadm/noderegistration_test.go

```diff
@@ -68,7 +69,8 @@ func TestNewNodeRegistration(t *testing.T) {
 					"node-labels": "test value one,test value two",
 				},
 			},
-			actual: kubeadm.NewNodeRegistration(
+			actual: kubeadm.SetNodeRegistration(
+				&kubeadmv1beta1.NodeRegistrationOptions{},
```

Comment:
As discussed, in a follow-up PR, add a test case where the base config has some values.

Reply:
ok will do.